CHANGES.rst

```diff
@@ -343,6 +343,8 @@ astropy.coordinates
 - Ensure that for size-1 array ``SkyCoord`` and coordinate frames
   the attributes also properly become scalars when indexed with 0.
   [#10113]
+- Delay download of leap-second table in ``earth_orientation.py`` until
```

Comment:
Needs empty line, I think...

Reply:
Ops, good catch! I'll add it if we decide to proceed with this PR.